### PR TITLE
[Components with more than 1 service] Refactor Portal Cache components from remaining Portal Cache modules

### PR DESCRIPTION
**Pull request by GabrielHenriquedev**

### CODE REVIEW

**Comment by catarineaguiar** [comment]
ci:test:clustering

**Comment by kyle-miho** [comment]
Failures unrelated

File: modules/apps/document-library/document-library-service/src/main/java/com/liferay/document/library/internal/repository/capabilities/PortalCapabilityLocatorImpl.java [hunk 1]
```diff
@@ -275,4 +276,19 @@ private void _clearLiferayDynamicCapabilities() {
 	@Reference
 	private TrashVersionLocalService _trashVersionLocalService;
 
+	private class PortalCapabilityCacheRegistryItem
+		implements CacheRegistryItem {
+
+		@Override
+		public String getRegistryName() {
+			return _getClassName();
```
Comment (rafaprax) [inline modules/apps/document-library/document-library-service/src/main/java/com/liferay/document/library/internal/repository/capabilities/PortalCapabilityLocatorImpl.java:284]: @GabrielHenriquedev, please hard-coded the className here using the `PortalCapabilityLocatorImpl.class.getName()` statement. Thx

File: modules/apps/portal-cache/portal-cache-impl/src/main/java/com/liferay/portal/cache/internal/dao/orm/FinderCacheImpl.java [hunk 2]
```diff
@@ -502,6 +491,22 @@ protected void activate(BundleContext bundleContext) {
...
+			CacheRegistryItem.class, new FinderCacheCacheRegistryItem(), null);
+	}
+
+	@Deactivate
+	protected void deactivate() {
+		_serviceRegistration.unregister();
+	}
+
+	private void _cleanCache() {
+		clearLocalCache();
```
Comment (rafaprax) [inline modules/apps/portal-cache/portal-cache-impl/src/main/java/com/liferay/portal/cache/internal/dao/orm/FinderCacheImpl.java:505]: Hey @GabrielHenriquedev, you do not need to create this private method. You can call the `clearCache` directly into the `FinderCacheCacheRegistryItem` class.

File: modules/apps/portal-cache/portal-cache-impl/src/main/java/com/liferay/portal/cache/internal/dao/orm/EntityCacheImpl.java [hunk 3]
```diff
@@ -291,6 +283,24 @@ protected void activate() {
...
+			CacheRegistryItem.class, new EntityCacheCacheRegistryItem(), null);
+	}
+
+	@Deactivate
+	protected void deactivate() {
+		_serviceRegistration.unregister();
+	}
+
+	private void _cleanCache() {
+		_notifyFinderCache(null, null, false);
```
Comment (rafaprax) [inline modules/apps/portal-cache/portal-cache-impl/src/main/java/com/liferay/portal/cache/internal/dao/orm/EntityCacheImpl.java:297]: Hey @GabrielHenriquedev, you do not need to create this private method. You can call the clearCache directly into the EntityCacheCacheRegistryItem class.

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Pull request closed without merging**